Source/FullScreenPokemon.d.ts

```diff
@@ -906,6 +906,21 @@ declare module FullScreenPokemon {
          */
         grass?: IWildPokemonSchema[];
 
+        /**
+         * Types of Pokemon that may be caught with the Old Rod.
+         */
+        oldRod?: IWildPokemonSchema[];
```

Comment:
Instead of having three separate attributes, have one `"fishing"` attribute with `"old": ...`, etc.